modules/diskq/dqtool.c

```diff
@@ -730,9 +730,7 @@ main(int argc, char *argv[])
   scratch_buffers_allocator_init();
   log_template_global_init();
   log_msg_registry_init();
-  log_tags_global_init();
```

Comment:
`log_tags_global_init()` is missing now. I think we should remove `log_msg_registry_init()` as well and call `log_msg_global_init()`.